--- rosapanel-1.6.0/panel.cpp.kmail~	2013-06-23 21:39:21.391808130 +0200
+++ rosapanel-1.6.0/panel.cpp	2013-06-23 21:42:50.988187815 +0200
@@ -202,7 +202,7 @@ void Panel::initRosa()
     args.append( KUrl("file:///usr/share/applications/kde4/kopete.desktop").url() );
     addApplet("rosaicon",args);
     args.removeAt(0);
-    args.append( KUrl("file:///usr/share/applications/mandriva-mozilla-thunderbird.desktop").url() );
+    args.append( KUrl("file:///usr/share/applications/kde4/KMail2.desktop").url() );
     addApplet("rosaicon",args);
     args.removeAt(0);
     args.append( KUrl("file:///usr/share/applications/kde4/amarok.desktop").url() );
