--- rosapanel-1.6.0/panel.cpp.omv~	2013-09-02 13:21:56.664613475 +0200
+++ rosapanel-1.6.0/panel.cpp	2013-09-02 13:23:25.187940003 +0200
@@ -187,9 +187,19 @@ void Panel::initRosa()
     m_layout->setMaximumSize(QSize(screenRect.width(),DEFAULT_PANEL_SIZE));
     m_layout->setMinimumSize(QSize(screenRect.width(),DEFAULT_PANEL_SIZE));
     
+#if 0
     if (!addApplet("rosastarter")) {
         addApplet("launcher");
     }
+#else
+    /* FIXME this is for 2013.0 beta 1 only: Add all 3 important launchers
+     * so we can get some feedback and decide which will remain by default
+     */
+    addApplet("launcher");
+    addApplet("homerunlauncher");
+    addApplet("rosastarter");
+    addApplet("lancelot_launcher");
+#endif
     
     QVariantList args;
     
