--- rosapanel-1.5.0/icon-applet/SmoothTasks/Applet.cpp.omv~	2013-06-07 20:07:32.570750957 +0200
+++ rosapanel-1.5.0/icon-applet/SmoothTasks/Applet.cpp	2013-06-07 20:11:29.527962775 +0200
@@ -383,6 +383,9 @@ void Applet::clear() {
 }
 
 void Applet::reload() {
+	KConfigGroup cg = config();
+	setUrl(cg.readEntry("Url", m_url));
+
 	TaskManager::TaskGroup *group = m_groupManager->rootGroup();
 	
 	if (group != m_rootGroup.data()) {
